### ISSUE TITLE
isparta instrumenter is not working with karma-coverage v5.0.0

### ISSUE THREAD

**Issue by martinmicunda** [karma-runner/karma-coverage#175]
I try get run `isparta` instrumenter with karma-coverage v5.0.0 but coverage code is still ES5 and not ES6 and my understanding is that we should not use karma-coverage fork from @douglasduteil as it mentioned in this [comment](https://github.com/douglasduteil/isparta/issues/38#issuecomment-128466597) by @Dignifiedquire. Has anybody try `isparta` with `karma-coverage` v5.0.0? Thanks

**Comment by onlywei** [comment]
@martinmicunda I think you mean `v0.5.0`. I also just tried with `v0.5.2` and I am still seeing ES5 and not ES6 code.

**Comment by martinmicunda** [comment]
@onlywei yeah I meant `v0.5.0` and even I am running the latest `v0.5.2` but the code is still ES5 and not ES6... maybe @Dignifiedquire and @douglasduteil could help here?

**Comment by martinmicunda** [comment]
@Dignifiedquire @douglasduteil is there any update on this issue? Thanks

**Comment by pewh** [comment]
+1

**Comment by jashatton** [comment]
@Dignifiedquire @douglasduteil @martinmicunda I was hoping to get this working as well, any chance of another temporary fork, or a know workaround? Thanks

**Comment by martinmicunda** [comment]
@jashatton I wrote a blog how to get run unit test with coverage.. Check it [here](http://martinmicunda.com/2015/10/19/how-to-start-writing-unit-tests-with-es6-angular-1x-and-jspm/)

**Comment by jashatton** [comment]
@martinmicunda  It is a great article, thank you for writing it.  It mirrors our set up and what brought me to this issue.  Did you ever get ES6 code to display in the coverage reports?

**Comment by martinmicunda** [comment]
@jashatton yeah it mentioned in the article.. You can see ES6 coverage report [here](http://martinmicunda.com/employee-scheduling-ui/test-reports/coverage/firefox/)

**Comment by jashatton** [comment]
@martinmicunda  OK I will look at the article and setup again and see what is different.

**Comment by jashatton** [comment]
@martinmicunda It looks like you are still using the Istanbul and Karma Coverage forks, maybe on a local repo you are accessing?  These aren't available through Github anymore.  Any suggestions for solutions will be great.  I will come back with an update if I find anything useful.

**Comment by jashatton** [comment]
Well, after a couple hours of reading and inferring into documentation and Stack Overflow anwsers, I figured out how to get branch versions of npm modules installed.  But, I have it working as well.  Thanks again for the article it was the catalyst for me to learn more about npm and have readable coverage reports.

**Comment by jsg2021** [comment]
@jashatton what modules did you switch to branches?

**Comment by onlywei** [comment]
Is there an update here?

**Comment by jashatton** [comment]
Wow, sorry I missed the notifications.  I am currently using what is below in my package.json
```javascript
devDependencies {
    "babel-core": "5.8.21",
    "gulp": "3.9.0",
    "gulp-jshint": "1.11.2",
    "gulp-sass": "2.0.2",
    "isparta": "3.5.0",
    "istanbul": "git+http://github.com/gotwarlost/istanbul/#source-map",
    "jasmine-core": "2.3.4",
    "jshint-stylish": "2.0.1",
    "jspm": "0.15.7",
    "karma": "0.13.9",
    "karma-babel-preprocessor": "5.2.2",
    "karma-chrome-launcher": "0.2.0",
    "karma-coverage": "git+https://github.com/douglasduteil/karma-coverage.git#next",
    "karma-jasmine": "0.3.6",
    "karma-jspm": "2.0.1-beta.1",
    "karma-junit-reporter": "0.3.8",
    "karma-ng-html2js-preprocessor": "0.2.0",
    "karma-phantomjs-launcher": "0.2.1",
    "karma-sourcemap-loader": "0.3.6",
    "phantomjs": "1.9.18",
    "shelljs": "0.5.1",
    "vinyl-source-stream": "1.1.0"
}
```
@jsg2021 I am referencing Github npm modules for karma-coverage and istanbul to get coverage reports that have usable sourcemaps.

If anyone could use more information certainly let me know, I will try to keep an eye out for notifications.

**Comment by RomanGotsiy** [comment]
Still don't work on v0.5.3
But works great with @douglasduteil fork

**Comment by blittle** [comment]
@RomanGotsiy how do you set it up with @douglasduteil's fork? Doesn't look like he has a release tar.gz file to install from npm.

**Comment by RomanGotsiy** [comment]
@blittle 
`npm install github:douglasduteil/karma-coverage#next --save-dev` does the trick.

You can check my [package.json](https://github.com/Rebilly/ReDoc/blob/master/package.json) and [karma.conf.js](https://github.com/Rebilly/ReDoc/blob/master/karma.conf.js)

**Comment by blittle** [comment]
@RomanGotsiy Thanks!

**Comment by AdamWillden** [comment]
+1

Be great if this could be fixed. Seems that the douglasteil branch cant use the coverage check option so either way I have to lose some functionality that I want :-(

I'm assuming this is a problem with karma-coverage and not isparta?

**Comment by otroboe** [comment]
+1
